pkg/controller/pravega/pravega_segmentstore.go

```diff
@@ -232,6 +232,9 @@ func MakeSegmentstoreConfigMap(p *api.PravegaCluster) *corev1.ConfigMap {
 
 	if p.Spec.ExternalAccess.Enabled {
 		configData["K8_EXTERNAL_ACCESS"] = "true"
+		serviceName := util.ServiceNameForSegmentStore(p.Name, 0)
+		serviceName = serviceName[0 : len(serviceName)-2]
+		configData["SERVICE_NAME"] = serviceName
```

Comment:
Adding this new variable breaks compatibility of Operator 0.5.0 with older Pravega versions. Why don't we just set env var POD_NAME to point to SSS Servicename instead?

Reply:
we cant set POD_NAME to SSS name, if we do that it in `init-kubernets.sh`  where it uses  `nodename=$( k8 "${ns}" "pods" "${podname}" ".spec.nodeName" )`  fails